### ISSUE TITLE
Help Making it darker

### ISSUE THREAD

**Issue by VinceMendneck** [ArshSB/DarkPDF#2]
hey man, awesome extension, it made me start looking into js, ive been trying to change the border to black so it all looks black, do you think it can be done? i tried messing with your code a bit but i can only change the full color of the page, no success in making it custom made, do you think it is possible? , thanks for your work.

**Comment by ArshSB** [comment]
Hey Vince, thank you for your feedback.

I am not sure what you mean by changing the border to black. Can you please elaborate?

**Comment by VinceMendneck** [comment]
of course, as you can see in the image i have uploaded, only the actual page is black, the pdf interface is light gray, almost white, i wonder if that is a way to make everything in the page black, except for the words of course, those would still be white or maybe even another color if fully customizable
![Untitled45454](https://user-images.githubusercontent.com/108556680/177203048-ba98c263-9748-4024-b335-3f58a52db078.png)

**Comment by ArshSB** [comment]
I understand now. I tried to fix this issue but it turned out to be much more work than I thought. Sadly, this issue is beyond my technical skill.

The way the dark mode for this extension works is that I apply a `div` element to the entire page that inverts the colors. I tried to use [this](https://pdf-lib.js.org/) library to get the PDF content width and height, so I can apply the `div` element only at those dimensions. However, the problem is that the moment you zoom in/out or click ≡ at top left corner, the dimensions change resulting in the dark mode being applied unevenly and randomly.

I also tried to gain access into the PDF document itself, so that I can view its content similar to HTML. Unfortunately, the entire PDF document is an embedded element to which we don't have access to.

![image](https://user-images.githubusercontent.com/47343364/177377353-1ca34ebc-3c58-445c-84ab-b814792525cc.png)

Good news though! Another user found a work around to this issue, but you will have to use a different PDF viewer:

![image](https://user-images.githubusercontent.com/47343364/177378014-4cc2da33-9ecf-4197-8c64-aa2e123ac4d6.png)

I hope this was helpful. I am glad you found my extension useful.
